### PR TITLE
add support for foreman_azure_rm v2

### PR DESCRIPTION
**Pull request by hlawatschek**
In foreman_azure_rm V2 some API endpoints have been changed.
tested with foreman-1.24.2  and foreman_azure_rm 2.0.7

### CODE REVIEW

**Comment by mdellweg** [comment]
Is it still working with the old one then?

**Comment by hlawatschek** [comment]
No, unfortunately not. But imo the "old" one has not been delivered as package by upstream anyway.

File: plugins/modules/foreman_compute_resource.py [hunk 1]
```diff
@@ -243,7 +248,7 @@ def get_provider_info(provider):
         return 'EC2', ['user', 'password', 'region']
 
     elif provider_name == 'azurerm':
-        return 'AzureRM', ['url', 'user', 'password', 'tenant', 'region']
+        return 'AzureRm', ['url', 'user', 'password', 'tenant', 'region', 'app_ident']
```
Comment (mdellweg) [inline plugins/modules/foreman_compute_resource.py]: should this 'url' then be removed?

Comment (mdellweg) [reply plugins/modules/foreman_compute_resource.py]: Also the capitalization should be adjusted in the other places as well.

**Pull request merged**